apps/server/src/modules/school/api/school.uc.ts

```diff
@@ -11,15 +13,18 @@ import {
 	SchoolSystemResponse,
 } from './dto/response';
 import { SchoolForLdapLoginResponse } from './dto/response/school-for-ldap-login.response';
+import { SchoolUserListResponse } from './dto/response/school-user.response';
 import { SchoolResponseMapper, SystemResponseMapper } from './mapper';
+import { SchoolUserResponseMapper } from './mapper/school-user.response.mapper';
```

Comment:
suggest to include SchoolUserResponseMapper to barrel file ./mapper (see line 17)

Reply:
fixed